src/dmtcp_launch.cpp

```diff
@@ -272,7 +272,11 @@ processArgs(int *orig_argc,
       shift;
     }
 #endif // ifdef HBICT_DELTACOMP
-    else if (s == "--new-coordinator") {
+    else if (s == "--file-socket-path") {
```

Comment:
Instead of asking the user for the file socket path, can't DMTCP generate one automatically?

Reply:
Users might not have permission if we chose a default location.  This option provides flexibility if we scale DMTCP and want these locations to be customized.  Where would you like to store it and call it?  Now I usually test it via `/tmp/dmtcp_network.socket`